tests/h/views/admin_users_test.py

```diff
@@ -185,7 +181,7 @@ def test_users_activate_redirects(pyramid_request):
     assert isinstance(result, httpexceptions.HTTPFound)
 
 
-users_delete_fixtures = pytest.mark.usefixtures('user_service', 'fake_delete_user')
+users_delete_fixtures = pytest.mark.usefixtures('user_service')
```

Comment:
Now that every test that’s marked with `@users_delete_fixtures` also includes `user_service` as a parameter, I think you could safely remove this.